models/issue_watch_test.go

```diff
@@ -48,9 +48,16 @@ func TestGetIssueWatchers(t *testing.T) {
 
 	iws, err = GetIssueWatchers(2)
 	assert.NoError(t, err)
-	assert.Equal(t, 1, len(iws))
+	// Watcher is explicit not watching
+	assert.Equal(t, 0, len(iws))
```

Comment:
```suggestion
	assert.Len(t, iws, 0)
```
😉 Only if you're also pushing other changes. Not that it will make much of a difference.